go.mod

```diff
@@ -51,17 +51,17 @@ require (
 	golang.org/x/sync v0.12.0
 	golang.org/x/term v0.30.0
 	golang.org/x/text v0.23.0
-	k8s.io/api v0.31.6
-	k8s.io/apiextensions-apiserver v0.31.6
-	k8s.io/apimachinery v0.31.6
-	k8s.io/apiserver v0.31.6
-	k8s.io/client-go v0.31.6
-	k8s.io/component-base v0.31.6
+	k8s.io/api v0.31.5
```

Comment:
this seems wrong

Reply:
it has been fixed with [2845829](https://github.com/vmware-tanzu/pinniped/pull/2285/commits/2845829fb4757347604b4bfbb94bca03f1765575)